qml/components-v-play/ActionMenuItem.qml

```diff
@@ -11,5 +11,7 @@ BVApp.MenuItem {
 
     signal clicked
 
-    onSelected: clicked()
+    onSelected: {
+        clicked()
```

Comment:
C'mon, you only made this change in another commit to
make the actual fix be a one liner!